modular_skyrat/modules/modular_implants/code/soulcatcher/handheld_soulcatcher.dm

```diff
@@ -27,10 +30,17 @@
 	if(linked_soulcatcher)
 		qdel(linked_soulcatcher)
 
+	interacting_mobs = null
+
+	for (var/mob/soul as anything in confirming_entry)
+		UnregisterSignal(soul, COMSIG_PARENT_QDELETING)
+		confirming_entry -= soul
+	confirming_entry = null
```

Comment:
if we are making this list null, why are we removing elements from it?

Reply:
mostly just because i was already iterating through it so i might as well